src/main/python/covata/delta/interfaces.py

```diff
@@ -64,3 +68,43 @@ def get_identity(self, requestor_id, identity_id):
         :return: the retrieved identity
         :rtype: dict
         """
+
+
+@six.add_metaclass(ABCMeta)
+class KeyStore(object):
+
+    @abstractmethod
+    def save(self, private_key, name):
+        """
+        Saves a private key object (encrypted) to keystore.
+
+        Saving the Private Cryptographic Key:
+
+        >>> keystore.save(private_key, identity_id + ".crypto.pem")
+
+        Saving the Private Signing Key:
+
+        >>> keystore.save(private_key, identity_id + ".signing.pem")
+
+        :param private_key: the private key object
+        :type private_key: :class:`RSAPrivateKey`
+        :param str name: the name of the .pem file to be written
+        """
+
+    @abstractmethod
+    def load(self, name):
+        """
+        Loads a private key instance from an encrypted .pem file
+        in the keystore.
+
+        Loading the Private Cryptographic Key:
```

Comment:
Why not two methods here - one for encryption key and one for signing key?

Like the Java counterpart: public PublicKey getPublicSigningKey(String identityId)